test/IdleMStableStrategy.js

```diff
@@ -0,0 +1,193 @@
+require("hardhat/config");
```

Comment:
Why is this file in this PR?

Reply:
The integration file got copied. It shouldn't have been there.
I have removed it.